api/spec/src/billing/invoices.tsp

```diff
@@ -0,0 +1,450 @@
+using TypeSpec.Http;
+using TypeSpec.OpenAPI;
+
+namespace OpenMeter.Billing;
+
+/**
+ * InvoicesAllCustomers is a collection of endpoints that allow invoice operations without prior
+ * knowledge of the customer ID.
+ */
+@route("/api/v1/billing/invoices")
+@tag("Billing (Experimental)")
+interface InvoicesAllCustomers {
```

Comment:
does the "all" stands for something specific?

Reply:
I have renamed the interfaces to:
- Invoices
- CustomerInvoices
- CustomerInvoice

It's a better seperatation.